docs/developer_setup.md

```diff
@@ -7,10 +7,19 @@ You will need one of the following
 - *recommended* - Having `nix` installed following these [instructions](https://nixos.org/download.html).
 - *alternative* - Having Rust installed via [rustup](https://www.rust-lang.org/tools/install) and configured to use the stable toolchain.
 
-### Build holochain
+### Set up Cachix
 
-If using Nix, start by opening a developer shell using `nix develop .#coreDev` at the root of the repository. This will take
-some time you run it.
+Getting Holochain's build tools from source takes a long time to build, so we provide a cache so you can download binaries for your system.
+
+If you installed Nix using Holochain's [getting started guide](https://developer.holochain.org/get-started/) then Cachix will have been set up for you. Otherwise you can set it up yourself by installing Cachix using their [install guide](https://docs.cachix.org/installation).
+
+You can then run `cachix use holochain-ci`.
+
+Now when you run `nix develop` or `nix build` commands, you should look out for the status line in your shell telling you that it's downloading pre-built binaries from `holochain-ci.cachix.org`.
+
+### Build Holochain
+
+If using Nix, start by opening a developer shell using `nix develop --override-input versions ./versions/weekly --override-input holochain . .#coreDev` at the root of the repository. This will take a while the first time you run it.
```

Comment:
technically these overrides shouldn't have any effect. i know i've seen omitting the versions override cause issues (without cachix set up (which i consider a bug)), however i think (and hope) that the holochain override doesn't have any (positive) effect here.

Reply:
Fair enough, I tend to leave them included but I haven't seen any negative consequences when I've omitted them. Shall I leave them out of this doc for simplicity?

Reply:
Well, the `--override-input holochain .` I would remove. The weekly input does have an effect

Reply:
> the `--override-input holochain .` I would remove. The weekly input does have an effect

SGTM